QuickBlox/quickblox-android-sdk
# feature/ANDSAMPLES-507

2. fixed Chat Info screen with updating current dialog from server
3. fixed Add Users screen with the same updating like in Chat Info screen

*Make sure that you wrote the tests for your proposed changes and the existing test was success.*

**Made/Proposed changes:**
*(Don’t use general words, describe changes in details)*
- added timestamp to system message about creating dialog
- fixed Chat Info screen with updating current dialog from server
- fixed Add Users screen with the same updating like in Chat Info screen

**How should this be manually tested?**
- Log in as users who have common group chat
-  As User 2 leave the group chat
-  As User 1 press "Add users to chat" button → Add users to chat screen appears
Result: User 2 is not in the group's participants list
- As User 1 press "Chat info" button →Chat info screen appears
Result: User 2 is not in the group's participants list

**Does the documentation need an update?**
documentation updates is not needed
